### PR TITLE
add rivescript@1.17.1 w/ npm auto-update

### LINKED ISSUE #10090: [Request] Add rivescript
**Library name:**rivescript
**Git repository url:**
https://github.com/aichaos/rivescript-js
**npm package url(optional):** https://www.npmjs.com/package/rivescript
**License(s):**
**Official homepage:**
https://www.rivescript.com
**Wanna say something? Leave message here:**
easy to use chatbot builder

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
close #10090, cc @kirsle

@pvnr0082t Please help me review this PR. Thank you
Pull request for issue: #10090
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/aichaos/rivescript-js
 * Official website (optional, not the repository): https://www.rivescript.com/
 * NPM package url (optional): https://www.npmjs.com/package/rivescript
 * License and its reference: MIT, ref: https://github.com/aichaos/rivescript-js/blob/master/LICENSE
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 13
   - Count of watchers: 136
   - Count of forks: 38
 * NPM download stats (optional):
   - Downloads in the last day: 36
   - Downloads in the last week: 174
   - Downloads in the last month: 805

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/rivescript/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "aiml",
+    "artificial intelligence",
+    "chatscript",
+    "buddyscript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aichaos/rivescript-js.git"
+  },
+  "homepage": "http://www.rivescript.com/",
```
Comment (pvnr0082t) [inline ajax/libs/rivescript/package.json]: @hare1039 We can use `https` instead of `http` because it support SSL.

Comment (hare1039) [inline ajax/libs/rivescript/package.json]: @pvnr0082t Sure!

Comment (hare1039) [inline ajax/libs/rivescript/package.json]: @pvnr0082t I've appended the `s` on the `http`. Please review this PR again. Thank you

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Good job @hare1039 @pvnr0082t @kennynaoh

**Pull request merged**